js/views/modals/Settings/Advanced/Server.js

```diff
@@ -0,0 +1,30 @@
+import app from '../../../../app';
```

Comment:
I have some comments for these individual views, but I'm holding them back because, as I mentioned above, I strongly recommend they be consolidated into one Advanced view.